doc/lspconfig.txt

```diff
@@ -500,9 +404,9 @@ is typically (in rough order):
 - the built-in client in Nvim core
 - nvim-lspconfig
 
-The first step in debugging is to test with a minimal configuration (such as
-`../test/minimal_init.lua`). Historically, many users problems are due to
-plugins or misconfiguration.
+The first step in debugging is to test with a minimal configuration:
+https://github.com/neovim/neovim/issues/new?assignees=&labels=bug%2Clsp&template=lsp_bug_report.yml(such as
```

Comment:
Link to core LSP bug report, which provides a minimal init.

Remove `minimal_init.lua` from this repo.

Reply:
Hi, I am trying to set up my lsp first time in neovim and I have to be honest, it is not straightforward to see a default key mappings for neovim lsp, any where, until I found this `minimal_init.lua`. And you the link in `lspconfig.txt`, is a link to the bug report creation page, not a link to the `minimal_init` in core. If we could point to the right link for a minimal setup that'll be much appreciated. Personally,  I think it does make sense to make it a README must-have to mention where's the minimal default key maps for any lsp. Thanks very much!

Reply:
> Link to core LSP bug report, which provides a minimal init.
> Hi, I am trying to set up my lsp first time in neovim and I have to be honest, it is not straightforward to see a default key mappings for neovim lsp, any where, until I found this `minimal_init.lua`. And you the link in `lspconfig.txt`, is a link to the bug report creation page, not a link to the `minimal_init` in core. If we could point to the right link for a minimal setup that'll be much appreciated. Personally,  I think it does make sense to make it a README must-have to mention where's the minimal default key maps for any lsp. Thanks very much!
> Remove `minimal_init.lua` from this repo.

Reply:
> Link to core LSP bug report, which provides a minimal init.
> 
> Remove `minimal_init.lua` from this repo.



Hi, I am trying to set up my lsp first time in neovim and I have to be honest, it is not straightforward to see a default key mappings for neovim lsp, any where, until I found this `minimal_init.lua`. And you the link in `lspconfig.txt`, is a link to the bug report creation page, not a link to the `minimal_init` in core. If we could point to the right link for a minimal setup that'll be much appreciated. Personally,  I think it does make sense to make it a README must-have to mention where's the minimal default key maps for any lsp. Thanks very much!

Reply:
> whether default key maps should be part of lspconfig

Nvim 0.11 added default mappings for LSP. It's not released yet but you can try it here: https://github.com/neovim/neovim/releases